api/design/catalog.go

```diff
@@ -69,4 +69,30 @@ var _ = Service("catalog", func() {
 		})
 	})
 
+	Method("CatalogError", func() {
+		Description("List the all type of errors of a catalog")
+		Security(types.JWTAuth, func() {
+			Scope("catalog:refresh")
+		})
+		Payload(func() {
+			Attribute("catalogName", String, "Name of catalog", func() {
+				Example("catalogName", "tekton")
+			})
+			Token("token", String, "JWT")
```

Comment:
`Example`? just to make sure openapi file doesn't have any random string as example